frontend/src/scenes/web-analytics/LiveMetricsDashboard/liveWebAnalyticsMetricsCharts.tsx

```diff
@@ -0,0 +1,156 @@
+import { ChartConfiguration } from 'lib/Chart'
+import { getSeriesColor, getSeriesColorPalette } from 'lib/colors'
+import { useChart } from 'lib/hooks/useChart'
+
+import { ChartDataPoint, DeviceBreakdownItem } from './LiveWebAnalyticsMetricsTypes'
+
+export const UsersPerMinuteChart = ({ data }: { data: ChartDataPoint[] }): JSX.Element => {
```

Comment:
I think we can have a `*PerSecond` as well, and maybe choose the value between unique users and pageviews, but we can do this in another iteration.